packages/angular/cli/src/package-managers/error.ts

```diff
@@ -35,3 +35,18 @@ export class PackageManagerError extends Error {
     super(message);
   }
 }
+
+/**
+ * Represents structured information about an error returned by a package manager command.
+ * This is a data interface, not an `Error` subclass.
+ */
+export interface ErrorInfo {
+  /** A specific error code (e.g. 'E404', 'EACCES'). */
+  readonly code: string;
```

Comment:
Is it worth to make this narrower than `string` to make it clearer what _kind_ of error codes to expect?